lib/cylc/scheduler.py

```diff
@@ -50,7 +51,8 @@
 from cylc.xtrigger_mgr import XtriggerManager
 from cylc.suite_events import (
     SuiteEventContext, SuiteEventError, SuiteEventHandler)
-from cylc.hostuserutil import get_host, get_user
+from cylc.hostuserutil import get_host, get_user, get_fqdn_by_host
+import cylc.scheduler_cli
```

Comment:
The main import here is `HostAppointer`? Should we just move it to a separate module, so we don't end up with this cyclic import?

Reply:
Will do.